src/ensemble.jl

```diff
@@ -3,6 +3,16 @@ function SciMLBase.EnsembleProblem(prob::OptimizationProblem, u0s::Vector{Vector
     return SciMLBase.EnsembleProblem(prob; prob_func, kwargs...)
 end
 
+function SciMLBase.EnsembleProblem(prob::OptimizationProblem, trajectories::Int; kwargs...)
```

Comment:
I'd just keep to the interface. This doesn't seem like a necessary shorthand to me.

Reply:
I'd like to provide an automated way of doing the solves with initial points sampled, it would be useful. I can probably do it another way if you are opposed to this, but do note that the other dispatch is already not conforming fully with the current interface of creating these problems

Reply:
it would be good to come up with such a shorthand at a higher level of the interface, since NonlinearProblem, NonlinearLeastSquaresProblem, etc. would want to do similar things.